src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2644,6 +2672,46 @@ static void _svgLoaderParserXmlOpen(SvgLoaderData* loader, const char* content,
 }
 
 
+static void _svgLoaderParserXmlStyle(SvgLoaderData* loader, const char* content, unsigned int length)
+{
+    char* tag;
+    char* name;
+    const char* attrs = nullptr;
+    unsigned int attrsLength = 0;
+
+    FactoryMethod method;
+    GradientFactoryMethod gradientMethod;
+    SvgNode *node = nullptr;
+
+    const char *buf = content;
+    unsigned buflen = length;
+
+
+    while (auto next = simpleXmlParseCSSAttribute(buf, buflen, &tag, &name, &attrs, &attrsLength)) {
+        if ((method = _findGroupFactory(tag))) {
+            //TODO - node->id ??? add additional var for svgnode?
```

Comment:
@mgrudzinska tip: that will be better if you add TVGLOG() for those missing tag items so that we can figure out problems later when unsupported svg data comes.

Reply:
done